ajax/dropdownTrackingDeviceType.php

```diff
@@ -36,9 +36,19 @@
 
 Session::checkLoginUser();
 
+// Read parameters
+$context  = $_POST['context'] ?? '';
+$itemtype = $_POST["itemtype"] ?? '';
```

Comment:
`itemtype` is mandatory (see L. 51), so this is wrong to set it to `''` here. `$isValidItemtype` should be conditionned on `$_POST['itemtyp']`presence.
Also, if it is missing, a warning should be logged IMHO

Reply:
Fixed in latest commit.